### PR TITLE
Arcade Jan 2023 Release blog post

### LINKED ISSUE #9342: MakeCode Arcade Jan Release Blog Post
Hi Galen,
Can you get this blog post prepped for release?
All the files should be here:  https://1drv.ms/u/s!AmMIW5Hxi0RtgaEd2pBJ0gUIi25CIA?e=8WTdaB
Just waiting on 2 links for the Code Validation section from @kiki-lee and @thsparks.
Hoping to get this out today or tomorrow.
Thanks!

### PR DESCRIPTION
**Pull request by ganicke**
Import, format, edit, adjust images, etc. for the Arcade Jan 2023 Release blog post.

Closes #9342

### CODE REVIEW

**Review by Jaqster — APPROVED** [review]
Thanks Galen!  Looks great!

File: docs/blog/arcade/update-jan-2023.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+ 
+![Color palette editing](/static/blog/arcade/update-jan-2023/color-palette.gif)
+
+## Code Validation
+
+Being able to automatically validate the correctness of code in tutorials has been one of the top requested features from educators. In this release, we’ve added some basic code validation capabilities that will check for certain blocks on the workspace.
+
+![Code validation checks](/static/blog/arcade/update-jan-2023/code-validation.gif)
+
+For tutorial authors who would like to add code validation in their tutorials, please see the documentation on how to do this here <insert link to code validation documentation>
```
Comment (ganicke) [inline docs/blog/arcade/update-jan-2023.md:67]: Links to validation content here?

File: docs/blog/arcade/update-jan-2023.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+To see code validation in action, check out these tutorials <link to tutorials>
```
Comment (ganicke) [inline docs/blog/arcade/update-jan-2023.md:69]: ... and here?

File: docs/blog/arcade/update-jan-2023.md (hunk 1 above)
Comment (Jaqster) [reply docs/blog/arcade/update-jan-2023.md:67]: Oh, no.  Sorry, we didn't get any documentation done in time.  Let's nix that whole sentence.

File: docs/blog/arcade/update-jan-2023.md (hunk 2 above)
Comment (Jaqster) [reply docs/blog/arcade/update-jan-2023.md:69]: Yes, here we can say "To see code validation in action, check out these tutorials:
- [Time Flies](http://arcade.makecode.com/#tutorial:/tutorials/froggy)
- [Collect the Clovers](http://arcade.makecode.com/#tutorial:/tutorials/collect-the-clovers)

**Comment by Jaqster** [comment]
Hi Galen - is this ready to merge?

**Comment by ganicke** [comment]
I'm ready if you're ready.

**Comment by Jaqster** [comment]
Yeah!  Thanks!

**Pull request merged**